lighthouse-cli/test/smokehouse/perf/expectations.js

```diff
@@ -79,7 +79,7 @@ module.exports = [
               {resourceType: 'font', requestCount: 2, size: '80000±1000'},
               {resourceType: 'script', requestCount: 3, size: '55000±1000'},
               {resourceType: 'image', requestCount: 2, size: '28000±1000'},
-              {resourceType: 'document', requestCount: 1, size: '2100±100'},
+              {resourceType: 'document', requestCount: 1, size: '2200±100'},
```

Comment:
went up b/c content-type header is now added for html docs

Reply:
oh right that's easy to forget that header cost is included in these